net-im/telegram-desktop-bin/telegram-desktop-bin-3.0.1-r1.ebuild

```
@@ -33,9 +33,11 @@ RDEPEND="
 S="${WORKDIR}/Telegram"
 
 src_install() {
-	exeinto /usr/lib/${PN}
-	doexe "Telegram"
-	newbin "${FILESDIR}"/${PN} "telegram-desktop"
+	newbin "Telegram" "telegram-desktop"
+
+	insinto /etc/tdesktop/
+	echo "/usr/bin/telegram-desktop" > "${WORKDIR}/externalupdater"
+	doins "${WORKDIR}/externalupdater"
```

Missing EPREFIX, and echo that writes to a file would need `|| die`, but you can skip the latter by simplifying with:
```suggestion
	newins - externalupdater <<<"${EPREFIX}/usr/bin/telegram-desktop"
```

---

Oh and fwiw it wouldn't hurt to add a bugref to explain this, just a ` #814062` at the end of the line is enough.

---

applied that suggestion with stdin instead of echo, not sure the bug tag would be needed. Commit messages have it, do not feel like duplication.